packages/jsii-config/lib/questions.ts

```diff
@@ -55,8 +55,28 @@ function buildQuestions(
     choices: Object.keys(schema.jsii.targets),
     default: Object.keys(currentTargets),
   };
+  const tsconfigPrompt: QuestionCollection = {
+    name: 'tsconfig',
+    message:
+      'Typescript config - should jsii generate a compatible tsconfig or do you want to manage it yourself',
+    type: 'list',
+    choices: [
+      'jsii-managed',
+      new Separator('tsconfig will be managed and generated by jsii for you'),
```

Comment:
This order reads weird

```suggestion
      new Separator('tsconfig will be managed and generated for you by jsii'),
```